### DISCUSSION TITLE
Site down?

### DISCUSSION THREAD

**Discussion by blairham** [z-shell/zi#311] [General]
Seems the site has been down for 11 hours now, is anyone monitoring it?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ss-o** [comment]
Hi @blairham :wave: 

We encountered unexpected issues during our migration to a new provider. However, we have resolved them now.

**Reply by blairham** [reply]
Sorry to hear that, but glad it's resolve that you for the update!
